src/main/java/com/gene42/test/util/Box.java

```diff
@@ -15,7 +15,7 @@
     private String name;
 
 
-    public Box(String name, double width, double height){
+    public Box(String name, double width, double height) throws Exception {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS00112)